mlflow/.env

```diff
@@ -1,3 +1,5 @@
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=password
 POSTGRES_DB=mlflow
+POSTGRES_PORT=5432
+MLFLOW_PORT=5000
```

Comment:
Remove this file from repository.

Reply:
deleted [c6e7654](https://github.com/mizaqq/housepricesml/pull/3/commits/c6e765427b57568b8622b9d4233846e7aa5050ea)